roles/hosted_engine_setup/defaults/main.yml

```diff
@@ -36,6 +36,8 @@ he_webui_forward_port: 6900  # by default already open for VM console
 he_reserved_memory_MB: 512
 he_avail_memory_grace_MB: 200
 
+engine_psql: /usr/share/ovirt-engine/dbscripts/engine-psql.sh
```

Comment:
Using an external script might be tricky to debug in case of changes/failure.
IMO going with https://github.com/oVirt/ovirt-ansible-collection/pull/452 is a more conservative approach.